### PR TITLE
add very basic readme

### PR DESCRIPTION
**Pull request by roo2**
Adds a tiny readme with instructions for getting up and running with testing/ developing the new users welcome tour.

It doesn't give much context but just enough to start developing/testing the modal

### CODE REVIEW

File: apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Nux Welcome Tour Modal
+
+A help tour to show new users some of the basics of using the editor.
+
+### Testing Instructions
+
+Enable by adding this line to your `0-sandbox.php`
+
+```
+    define('SHOW_WELCOME_TOUR', true);
```
Comment (ramonjd) [inline apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/README.md]: Thanks a lot for thinking of this. I agree we should have a README with some basic information about what the feature does. 👍 

The testing instructions, assuming we'll eventually turn this version of the Welcome Tour on for everyone, will be outdated.

I wonder if we could remove the references to `0-sandbox.php` and ` define('SHOW_WELCOME_TOUR', true);`, but keep the line with the line to the Github PR (which will be still useful for testing existing sites).

What do you think @autumnfjeld ?

Comment (roo2) [reply apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/README.md]: ok, I just took that section out and left the link to the PR with test instructions

**Review by ramonjd — APPROVED** [review]
A good start! Thanks for thinking of this :)

File: apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/README.md (hunk 1 above)
Comment (autumnfjeld) [reply apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/README.md]: This is great @roo2 .  +1 for thinking about the little important things like README!

**Review by autumnfjeld — APPROVED** [review]

**Pull request merged**